extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -208,6 +208,25 @@ export class ProjectsController {
 	public async buildProject(context: Project | dataworkspace.WorkspaceTreeItem): Promise<string> {
 		const project: Project = this.getProjectFromContext(context);
 
+		const telemetryProps: Record<string, string> = {};
+
+		let databaseSource: string | undefined;
+		let databaseSourceValues = project.getRawDatabaseSourceValues();
+
+		for (let databaseSourceIndex = 0; databaseSourceIndex < databaseSourceValues.length; databaseSourceIndex++) {
+			if (utils.IsWellKnownValue(databaseSourceValues[databaseSourceIndex])) {
```

Comment:
Currently this is going to result in a string that only has one or more `dsct-oracle-to-ms-sql` in it (`dsct-oracle-to-ms-sql;dsct-oracle-to-ms-sql;dsct-oracle-to-ms-sql`...)

I'd suggest reducing that down so that duplicates aren't added (you can use Set to do this easily after getting the full array of database source values)